src/reconstruction/modifications/enzymes.jl

```diff
@@ -17,3 +17,20 @@ giving a [`EnzymeConstrainedModel`](@ref). The arguments are forwarded to
 """
 with_enzyme_constrained(; kwargs...) =
     model -> make_enzyme_constrained_model(model; kwargs...)
+
+"""
+$(TYPEDSIGNATURES)
+
+Specifies a model variant that adds a pseudoribosome to a model. Args and kwargs
+are forwarded to [`add_pseudoribosome`](@ref).
+"""
+with_pseudoribosome(args...; kwargs...) =
+    model -> add_pseudoribosome(model, args...; kwargs...)
+
+"""
+$(TYPEDSIGNATURES)
+
+Specifies a model variant that overwrites the current isozymes associated with the
+model through calling [`add_isozymes`](@ref).
+"""
+with_isozymes(args...) = model -> add_isozymes(model, args...)
```

Comment:
this only works on the ObjectModel now but whatevs; eventually we're going to generalize that to all models (with the wrappers) and this code stays intact.

As the only change pls forward kwargs too for future compat (even though there are none now). I'd even avoid having any positional args in the first place, luckily here it is kinda intuitive -- here it says you're adding isozymes so the only arg is likely gonna be the isozymes. :D